### ISSUE TITLE
Integrate translation service

### ISSUE THREAD

**Issue by erri120** [Nexus-Mods/NexusMods.App#598]
Part of the reason we're using resource files, is because many translation services support them. With #330 being done, we should look at integrating some translation service to crowdsource the translation for languages we don't support. Here are some services I found after a quick search:

- [Weblate](https://weblate.org/en/)
- [Crowdin](https://crowdin.com/)
- [Transifex](https://www.transifex.com/)

**Comment by Al12rs** [comment]
MO2 uses transifex. I don't have other comparison points to reference so I can't really comment on it too much. There are some pain points for maintenance, with maintainers having to approve new translators and new reviewers.  I didn't interact too much with it.

**Comment by DioKiriie** [comment]
Yeah, I was part of MO2 Translation at Transifex and Valve's projects at Crowdin. The last one definitely has better user experience as a translator.

**Comment by LukeNexusMods** [comment]
We need to pick a translation service and then use it. 

Investigate what we want, how to integrate the servers.

**Comment by LukeNexusMods** [comment]
Check which translation service is the best?
